xbmc/utils/URIUtils.cpp

```diff
@@ -924,7 +924,7 @@ bool URIUtils::IsFTP(const std::string& strFile)
          IsProtocol(strFile, "ftps");
 }
 
-bool URIUtils::IsHTTP(const std::string& strFile)
+bool URIUtils::IsHTTP(const std::string& strFile, const bool& bTranslate /* = false */)
```

Comment:
It does not make much sense to specify simple types as **const** ref param. Just use `bool`by val here.